app/controllers/boxes_controller.rb

```diff
@@ -13,6 +13,7 @@ def sync_all
     EventBus.publish(:box_sync_all_requested)
 
     Current.tenant.boxes.sync_all
+    redirect_back_or_to message_threads_path, notice: t("boxes.sync_all_running")
```

Comment:
ked je to event `:box_sync_all_requested` tak aj tu by som cakal 
```suggestion
    redirect_back_or_to message_threads_path, notice: t("boxes.sync_all_requested")
```